tools/picard/picard_CollectSequencingArtifactsMetrics.xml

```diff
@@ -0,0 +1,53 @@
+<tool id='picard_artifact_metrics' name='Picard Collect Sequencing Artifact Metrics'>
+  <description>writes 5 tables containing Phred-scaled error rates substitution details and summaries</description>
+  <!--<requirements>
+    <requirement type="package">picard</requirement>
+  </requirements>-->
+  <command>
+    <![CDATA[
+    
+    ##set up input files
+
+    #set $reference_fasta_filename = "/Users/aerijman/Documents/projects/galaxy/tool-data/hg38/seq/hg38.fa"
+    
+    picard -Xmx4048m
+    CollectSequencingArtifactMetrics
+    I='${sam_bam}'
+    O='${output_name}'
+    R='/Users/aerijman/Documents/projects/galaxy/tool-data/hg38/seq/hg38.fa' 
```

Comment:
probably should use something [like this](https://github.com/galaxyproject/tools-iuc/blob/64305ae905dc79e44b478203000add73598bcca1/tools/picard/picard_ValidateSamFile.xml#L23) for the reference genome.